hedera-node/hedera-token-service/src/main/java/com/hedera/node/app/service/token/AliasUtils.java

```diff
@@ -78,24 +78,19 @@ public static Bytes extractEvmAddress(@Nullable final Key key) {
 
     /**
      * Given some alias, determine whether it is an "entity num alias". If the alias is exactly 20 bytes long, and
-     * if its initial bytes match the entity prefix, then it is an entity num alias.
+     * if its initial bytes match the {@link #ENTITY_NUM_ALIAS_PREFIX}, then it is an entity num alias.
      *
      * <p>Every entity in the system (accounts, tokens, etc.) may be represented within ethereum with a 20-byte EVM
      * address. This address can be explicit (as part of the alias), or it can be based on the entity ID number. When
-     * based on the entity number, the first 12 bytes represent the shard and alias, while the last 8 bytes represent
-     * the entity number. When shard and realm are zero, this prefix is all zeros, which is why it is sometimes known as
-     * the "long-zero" alias.
+     * based on the entity number, the first 12 bytes will be defined to be zero which indicates the current networks
+     * shard and realm, while the last 8 bytes represent the entity number. Because the shard and realm are zero, this
+     * prefix is all zeros, which is why it is sometimes known as the "long-zero" alias.
```

Comment:
To check my understanding, I read here that a prefix of all zeros indicates the current network's shard/realm is `0.0`...but isn't the whole point of this PR to _not_ encode shard/realm in this alias? Seems like a contradiction to me, and one that could easily cause bugs

Reply:
What I was trying to indicate in this comment is that we would only recognize addresses with the first 12 bytes as "long zero" addresses.  When we encounter such an address, the entity id translated from this address should be `<local shard>.<local realm>.<entity number>`

I will update the comment to help clarify this intent.